docs/Foundation.md

```diff
@@ -1,29 +1,3 @@
 Colors, spacing, and typography are the building blocks of the design system and are often referred to as design tokens. Design tokens are named entities that store visual design attributes.
 
-Those values are accessible in the app via [ThemeProvider](https://github.com/component-driven/component-driven-development/blob/master/src/ThemeProvider.js) and the `theme` object. To ensure consistency, always use values from the `theme` object, instead of hardcoded values.
```

Comment:
You think this isn't useful anymore?

Reply:
This was moved to the workshop section since it's not useful on that page (design tokens documentation)